package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sum.cumo/vue-datepicker",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
Looks like while merging your fork with the master there was something wrong.
Those commits should not be around in this PR.